Ginger/Ginger/Run/RunSetActions/ExportResultsToALMConfigPage.xaml.cs

```diff
@@ -315,5 +319,15 @@ private void SetTestLevelComboBoxList(RunSetActionBase.eRunAt runAt)
                 }
             }
         }
+
+        private void SearchChildEntityByName_Unchecked(object sender, RoutedEventArgs e)
+        {
```

Comment:
Do we need this method if its body is empty?

Reply:
Removed